packages/react-i18n/README.md

```diff
@@ -90,7 +90,7 @@ The most commonly-used feature of the `@shopify/react-i18n` library is looking u
 Translations are provided using two keys in the `withI18n` decorator:
 
 - `fallback`: a translation file to use when translation keys are not found in the locale-specific translation files. These will usually be your English translations, as they are typically the most complete.
-- `translations`: a function which takes the locale and returns one of: nothing (no translations for the locale), a dictionary of key-value translation pairs, or a promise of one of the above. Alternatively, you can pass an object where the keys are locales, and the values are either translation dictionaries, or promises for translation dictionaries.
+- `translations`: a function which takes the locale and returns one of: nothing (no translations for the locale), a dictionary of key-value translation pairs, or a promise of one of the above. The `translations` function can also throw and `react-i18n` will handle the situation gracefully. Alternatively, you can pass an object where the keys are locales, and the values are either translation dictionaries, or promises for translation dictionaries.
```

Comment:
Can you make sure we have in here somewhere the recommendation that translations are inside a collocated `translations` directory, and that this pattern gets preferential treatment with the Babel plugin? I think we need to reinforce it a bit since it is on the magical side